.rubocop.yml

```diff
@@ -1,32 +1,39 @@
 Rails/CreateTableWithTimestamps:
   Enabled: false
 Rails/HasManyOrHasOneDependent:
-  Enabled: false
+  Enabled: true
+# if a codebase is on an older version on rails override this in the local rubocop.yml
 Rails/HttpPositionalArguments:
-  Enabled: false
+  Enabled: true
 Rails/LexicallyScopedActionFilter:
   Enabled: false
 Rails/Presence:
-  Enabled: false
+  Enabled: true
 Rails/FilePath:
   EnforcedStyle: arguments
-# We should enable this later but there is currently a bug in rubocop which
-# breaks this
-# Rails/RequestReferer:
-#   EnforcedStyle: referrer
-#
-# Also only allow update_all because it's fast.
-# Rails/SkipsModelValidations:
-#   Blacklist:
-#     - decrement!
-#     - decrement_counter
-#     - increment_counter
-#     - toggle!
-#     - touch
-#     - update_attribute
-#     - update_column
-#     - update_columns
-#     - update_counters
-
+Rails/RequestReferer:
+  EnforcedStyle: referrer
+# these paths are self explanatory in a Rails context and do not class comments
+Style/Documentation:
+  Exclude:
+    - 'db/migrate/*'
```

Comment:
should we add models and controllers here as well?

Reply:
I think having some short documentation on what each model represents might be beneficial to people unfamiliar with the code base, sometimes they are straightforward but other times could use a quick tweet about what it represents.